src/common/exif.h

```diff
@@ -82,7 +83,8 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
 int dt_exif_xmp_write(const int imgid, const char *filename);
 
 /** write xmp packet inside an image. */
-int dt_exif_xmp_attach_export(const int imgid, const char *filename, void *metadata);
+int dt_exif_xmp_attach_export(const int imgid, const char *filename, void *metadata,
+    dt_develop_t *dev, dt_dev_pixelpipe_t *pipe);
```

Comment:
looks wrong to have the dev & pipe used on `exif.h`.

Reply:
It looks weird, bu we need those to apply the dt_transform functions on the regions. Maybe I should add a comment explaining the reasoning?